HomeOxygenTherapy/R4/files/HomeOxygenTherapyPrepopulation-0.1.0.cql

```diff
@@ -82,37 +58,41 @@ concept "Intent Original Order": { "Intent Original Order code" } display 'Origi
 
 parameter device_request DeviceRequest
 
-// Relevant Diagnosis Code Lists
-define "Pulmonary_hypertension_Codes": { "I27.0", "I27.2" }
-
 // Observation code lists
+
+//Arterial Oxygen Codes
 define "Arterial_oxygen_saturation_Codes": { "59408-5" }
 define "Arterial_partial_pressure_of_oxygen_Codes": { "2703-7" }
 define "Arterial_oxygen_saturation_during_exercise_Codes": { "89276-0" }
 
-define "Hematocrit_lab_test_Codes": { "20570-8", "31100-1", "32354-3", "41654-5", "41655-2", "4544-3", "4545-0", "71829-6", "71830-4", "71832-0", "71833-8" }
-define "Hematocrit_Codes": {"31100-1"}
+//Hematocrit Code
+define "Hematocrit_lab_test_Codes": {"32354-3"}
```

Comment:
Why did you remove all of the other hematocrit lab test codes and are now only using one?

Reply:
The original list of hematocrit codes was based off of this Value Set: [2.16.840.1.113762.1.4.1045.114](https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1045.114/definition) (not one of our Value Sets). I spoke with Carmela, and she said that the only hematocrit value that was truly relevant in this context is Hematocrit of Arterial blood (32354-3). If you look at the HOT Lab Value Set from the spreadsheet [[2.16.840.1.113762.1.4.1219.106](https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1219.106/definition)], it only includes this one hematocrit code.